docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -11,15 +11,11 @@ When scanning with Semgrep in CI, findings automatically populate in Semgrep Clo
 
 If you run the alternate job and it fails with a "resource not accessible by integration" error, there are two possible causes.
 
-## Your repository is private
+## Your repository's default workflow permissions are set to read-only
 
-Findings from Semgrep can only populate in the GitHub Advanced Security Dashboard if the repository is public. Otherwise, the job does not have sufficient permissions to write to the dashboard regardless of any permissions set at the workflow or job level.
+Repository-level workflow permissions are set to `read-only` (default) unless they've previously been changed. Use of the `permissions` key within the workflow file does not override this setting.
```

Comment:
I'm a little bit confused about this statement as well. Below you say that for more granular permissions you can set the permissions at the workflow or job level, but here you're saying you can't do that. Which one is true, or are these not conflicting but I'm misunderstanding?